website/docs/docs/build/incremental-microbatch.md

```diff
@@ -8,7 +8,9 @@ id: "incremental-microbatch"
 
 :::info Microbatch
 
-The `microbatch` strategy is available in beta for [dbt Cloud Versionless](/docs/dbt-versions/upgrade-dbt-version-in-cloud#versionless) and dbt Core v1.9. We have been developing it behind a flag to prevent unintended interactions with existing custom incremental strategies. To enable this feature, [set the environment variable](/docs/build/environment-variables#setting-and-overriding-environment-variables) `DBT_EXPERIMENTAL_MICROBATCH` to `True` in your dbt Cloud environments or wherever you're running dbt Core.
+The `microbatch` strategy is available in beta for [dbt Cloud Versionless](/docs/dbt-versions/upgrade-dbt-version-in-cloud#versionless) and dbt Core v1.9. 
+
+If you use a custom microbatch macro, set a [distinct behavior flag](/reference/global-configs/behavior-changes#custom-microbatch-strategy) in your `dbt_project.yml` to enable batched execution. If you don't have a custom microbatch macro, you don't need to set this flag as dbt will handle microbatching automatically once you set the [microbatch strategy](#how-microbatch-compares-to-other-incremental-strategies) in your project.
```

Comment:
Can we change 
`once you set the [microbatch strategy](#how-microbatch-compares-to-other-incremental-strategies) in your project.`
to something like
`for any model using the [microbatch strategy](#how-microbatch-compares-to-other-incremental-strategies).`

I worry that the current wording might be confusing, as it implies one sets the microbatch strategy for the project instead of models.

Reply:
good call, thanks for that!